client/src/pages/CreateUserProfile.js

```diff
@@ -423,21 +434,23 @@ const CreateProfile = ({ email, firstName, lastName, history }) => {
               value="I agree to the Privacy Policy"
               onChange={handleInputChangePrivacy}
             >
-              By signing up, I agree to the{" "}
-              <StyledUnderlineLink onClick={showPrivacyPolicyModal}>
-                Privacy Policy
-              </StyledUnderlineLink>
+              <Trans i18nKey="profile.common.agreePrivacy">
+                <StyledUnderlineLink onClick={showPrivacyPolicyModal}>
+                  Privacy Policy
+                </StyledUnderlineLink>
+              </Trans>
             </StyledCheckbox>
             <WhiteSpace />
             <StyledCheckbox
               style={{ fontSize: "1.2rem" }}
               value="I agree to the Terms and Conditions"
               onChange={handleInputChangeConditions}
             >
-              By signing up, I agree to the{" "}
-              <StyledUnderlineLink onClick={showTermsConditionsModal}>
-                Terms and Conditions
-              </StyledUnderlineLink>
+              <Trans i18nKey="profile.common.agreeTerms">
+                <StyledUnderlineLink onClick={showTermsConditionsModal}>
+                  Terms and Conditions
```

Comment:
We should definitely translate these two. Terms & Conditions and Privacy policy.

Reply:
I think these two are already translated, since they are put in Trans component by i18next.